packages/client/src/components/interface/Navigation.tsx

```diff
@@ -250,7 +250,7 @@ export const NavigationView = (props: IFullProps) => {
     ? match.params.tabId
     : activeMenuItem
     ? activeMenuItem
-    : 'review'
+    : !window.location.href.includes('recordAudit') && 'review'
```

Comment:
Could we make this part a bit clearer? If I understand correctly, the purpose of this is to disable the tabs altogether when we're in the recordAudit view, right? 

Would it make sense to do this so, that in the record audit view, `<Navigation />` component would be given a property such as `deselectAllTabs={true}`, which would effectively do the same thing you're doing here. That way we'd build less magic inside the Navigation component and would keep the control in the component using Navigation